src/main/features/paid-in-full/views/date-paid.njk

```diff
@@ -0,0 +1,21 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, errorSummary, saveAndContinueButton %}
+{% from "timelineFragment.njk" import timelineFragment %}
+{% from "date.njk" import dateInput %}
+
+{% set heading = ('When did you receive the money?') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <form method="post">
+        <input type="hidden" name="known" value="true"/>
+        {{ csrfProtection(csrf) }}
+
+        <p>{{ t('Enter the exact date the money was clear in your account if you were paid by cheque or bank transfer.') }}</p>
+        {{ dateInput(form = form, name = 'date', hint = 'For example, 10 9 2018') }}
```

Comment:
Example in hint should not be hard coded

Reply:
Now fixed to be 2 weeks ago

Reply:
`10 9 2018` won't be two weeks ago in few months time. This need to be generated

Reply:
I've pushed the change to make it dynamic and calculate the example 14 days before today

Reply:
Thanks